src/Interface.jl

```diff
@@ -60,7 +58,7 @@ end
 macro forward_func(ex, fs)
     T, field = ex.args[1], ex.args[2].args[1]
     T = esc(T)
-    fs = isexpr(fs, :tuple) ? map(esc, fs.args) : [esc(fs)]
+    fs = Meta.isexpr(fs, :tuple) ? map(esc, fs.args) : [esc(fs)]
```

Comment:
Agreed. I think we also need to `import Base.Meta` ?

Reply:
Weirdly not - it just works on 0.6 and nightly. I guess Meta is automatically imported, but Meta.isexpr is not exported by Meta?